resources/sts/4.11/sts_installer_privatelink_permission_policy.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:ModifyVpcEndpointServiceConfiguration",
+		    "route53:ListHostedZonesByVPC",
+		    "route53:CreateVPCAssociationAuthorization",
+		    "route53:AssociateVPCWithHostedZone",
+		    "route53:DeleteVPCAssociationAuthorization",
+		    "route53:DisassociateVPCFromHostedZone",
```

Comment:
These are new too, there aren't in the old installer policy so I'm not sure why they are required now?

Reply:
Removed and testing without these. Do you know of any other permissions I can move into the privatelink policy from the core policy? There's only one left so it feels empty now :laughing: